ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileResource.java

```diff
@@ -33,20 +35,20 @@
     public static final List<Archiver> ARCHIVERS = Arrays.asList(new TarArchiver(),
             new TarArchiver(ReflectionBasedStreamFactory.create("java.util.zip.GZIPOutputStream", "application/gzip", ".gz")),
             new ZipArchiver());
+    private static final String HDT_CONTENT_TYPE = "application/vnd.hdt";
 
     public static DumpFileResource create(int domainId, String[] resourceUriTemplates, String[] accessUriTemplates,
             Graph[] graphs, Predicate<Request> predicate, Lang lang, CompressionStreamFactory compression, Archiver archiver) {
         DumpFileBuilder builder = new DumpFileBuilder(domainId, resourceUriTemplates, accessUriTemplates, graphs,
                 lang, compression);
         try {
-            File dumpFile = builder.build();
-            String contentType = builder.buildContentType();
+            File dumpFile = builder.buildHDT();
+            String contentType = HDT_CONTENT_TYPE;
```

Comment:
This means that _all_ dump file nodes _always_ use HDT. That is not what we want to have, is it? It would be better to have it as an additional option (like the other serializations).